packages/components/src/form/docs/Form.stories.mdx

```diff
@@ -37,7 +37,13 @@ A form can stack fields vertically.
 
 <Preview>
     <Story name="vertical layout">
-        <Form fluid width="25%">
+        <Form fluid
```

Comment:
If you set this one as fluid, then should we set all the problematic examples also as fluid? Feels weird though to have all the samples as fluid.

Maybe we could add some hacky CSS only for the docs to update our input min width CSS variables for our small breakpoints. This hacky CSS would be specific to the component and won't alter the components in apps.

Same thing could be done for tiles and other components with a problematic min width.

Reply:
Well the core issue doesn't happens in our apps, only in SB because of the code block style.